streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsKafkaClient.java

```diff
@@ -280,26 +296,32 @@ private String getAnyReadyBrokerId() {
         return ensureOneNodeIsReady(nodes);
     }
 
-    private ClientResponse sendRequest(final ClientRequest clientRequest) {
+    /**
+     * @return the response to the request
+     * @throws TimeoutException if there was no response within {@code request.timeout.ms}
+     * @throws StreamsException any other fatal error
+     */
+    private ClientResponse sendRequestSync(final ClientRequest clientRequest) {
         try {
             kafkaClient.send(clientRequest, Time.SYSTEM.milliseconds());
-        } catch (final Exception e) {
+        } catch (final RuntimeException e) {
```

Comment:
Any reasons we want to change to RTE here?

Reply:
As `send()` does not throw any checked exception, it think it's cleaner code to catch `RuntimeException`